src/forecast.coffee

```diff
@@ -271,7 +278,7 @@ module.exports = (robot) ->
   fetchForecast = (callback) ->
 
     forecastKey = process.env.HUBOT_FORECAST_KEY
-    exclude = 'hourly,daily,flags'
+    exclude = 'flags'
```

Comment:
The manual forecast uses the info in hourly/daily flags.  Right now it uses previously cached data, but if you request several hours after your forecast window the data would be old.  If there's nothing in the last_json object, it would make a manual request itself